make/config.mk

```diff
@@ -124,6 +124,7 @@ USE_MKLDNN = 0
 
 USE_NGRAPH = 0
 USE_NGRAPH_IE = 0
+USE_NGRAPH_GPU = 0
```

Comment:
I didn't see this been used anywhere.

Reply:
It is used by `ngraph.mk` which is included by the Makefile and is pulled in via a git submodule from the ngraph-mxnet-bridge repo. c.f. https://github.com/NervanaSystems/ngraph-mxnet-bridge/blob/master/ngraph.mk#L119